timer_scheduler.js

```diff
@@ -14,7 +15,12 @@ function handle_timeout() {
   var queues = get_queues(queue_daily, queue_by_hour);
   for(var i = 0; i < queues.length; i++) {
     utils.logDebug("going to fetch expired event from " + queues[i]);
-    expire_events(queues[i]);
+    if(work_client.command_queue_length > 10 || work_client.offline_queue_length > 10) {
```

Comment:
there are are on flights request, we skip this round of scanning of expired entries. thus giving more time for redis to handle this pending requests.